scripts/delete-federation.sh

```diff
@@ -28,15 +28,16 @@ NS="${FEDERATION_NAMESPACE:-federation-system}"
 PUBLIC_NS=kube-multicluster-public
 NAMESPACED="${NAMESPACED:-}"
 
-KF_NS_ARG=
+KF_NS_ARG="--federation-namespace=${NS} "
 if [[ "${NAMESPACED}" ]]; then
-  KF_NS_ARG="--federation-namespace=${NS} --registry-namespace=${NS}"
+  KF_NS_ARG+="--registry-namespace=${NS}"
 fi
 
 # Unjoin clusters by removing objects added by kubefed2.
 HOST_CLUSTER="$(kubectl config current-context)"
 JOINED_CLUSTERS="$(kubectl -n "${NS}" get federatedclusters -o=jsonpath='{range .items[*]}{.metadata.name}{" "}{end}')"
 for c in ${JOINED_CLUSTERS}; do
+  echo ./bin/kubefed2 unjoin "${c}" --host-cluster-context "${HOST_CLUSTER}" --remove-from-registry --v=2 ${KF_NS_ARG}
```

Comment:
Even if this were desirable (and I don't think it is - a user can use `bash -x` to invoke the script if they want to debug), duplicating a command entirely like this represents a maintenance burden.  Please remove.

Reply:
done